src/modules/commander/HealthAndArmingChecks/checks/batteryCheck.cpp

```diff
@@ -222,7 +222,7 @@ void BatteryChecks::checkAndReport(const Context &context, Report &reporter)
 						    log_level, "Low battery");
 
 			if (reporter.mavlink_log_pub()) {
-				mavlink_log_emergency(reporter.mavlink_log_pub(), "Low battery\t");
+				mavlink_log_warning(reporter.mavlink_log_pub(), "Low battery\t");
```

Comment:
You're probably right, I should have done that already here when I had the opportunity because I started to distinguish:
https://github.com/PX4/PX4-Autopilot/commit/7e4760587153535f21a403de855d9e158c0764d3#diff-1f54f0a80d365c804b5828d657946655fd70a397a3adf52102015ea31fea7a7bR225
The reason I didn't was just to not change that part trying to not break the user experience I assumed was tested before.